c2corg_ui/static/js/map/map.js

```diff
@@ -997,7 +1012,7 @@ app.MapController.prototype.resetFeature = function() {
  */
 app.MapController.prototype.canDelete = function() {
   return this.edit && this.getVectorLayer_().getSource().getFeatures().length > 0
-      && this.initialGeometry_ && this.initialGeometry_['geom'];
+  && this.initialGeometry_ && this.initialGeometry_['geom'];
```

Comment:
That looks more readable to me to keep the indentation: this way it's obvious the second line is the continuation of the first one.